.github/workflows/mirror.yml

```diff
@@ -0,0 +1,95 @@
+name: Mirror Distributions
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '00 1 * * 1-5'
+
+jobs:
+  get-unmirrored-node-versions:
+    name: Get unmirrored Node.js versions
+    runs-on: ubuntu-22.04
+    outputs:
+      versions: ${{ steps.get-unmirrored-versions.outputs.versions }}
+    steps:
```

Comment:
Would https://github.com/Swatinem/rust-cache help improve the job time for this and the other `get-unmirrored-*` job? (The compile currently takes 2 mins)

Reply:
Ah yeah. I didn't even notice this. I guess I copy-pasted this from something older =P.

Reply:
Yeah, definite improvement: https://github.com/heroku/buildpacks-nodejs/actions/runs/5070462443/jobs/9105527006 (under 30 seconds now).